src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -177,6 +194,31 @@ var DOMPropertyOperations = {
   },
 
   /**
+   * Creates markup for an SVG property.
+   *
+   * @param {string} name
+   * @param {*} value
+   * @return {string} Markup string, or empty string if the property was invalid.
+   */
+  createMarkupForSVGAttribute: function(name, value) {
+    if (!isAttributeNameSafe(name) || value == null) {
+      return '';
+    }
+    var propertyInfo = DOMProperty.properties.hasOwnProperty(name) ?
+        DOMProperty.properties[name] : null;
```

Comment:
nit: 2 space indent (you have a couple of this exact 2 lines doing the same thing - maybe have a helper method?)